### DISCUSSION TITLE
Remote development for Helix

### DISCUSSION THREAD

**Discussion by AshleySchaeffer** [helix-editor/helix#7755] [Ideas]
Developing on remote systems seems to be more common these days. 

What does this story look like for Helix? 

Kakoune implements a client-server model which I imagine lends well to this approach, and projects like distant.vim seem to be trickling out (https://github.com/chipsenkbeil/distant.nvim).

I think there's value in doing something beyond relying on SSH to facilitate this as it could open the door to collaborative functionality, where multiple clients can work within the same Helix instance.

**Comment by razcore-rad** [comment]
A colleague of mine just pointed me to https://zed.dev and it looks like an interesting project. If it weren't GUI I'd jump on it immediately, but yea one of the stronger points I saw in the features is this collaborative remote work.

**Comment by kirawi** [comment]
See https://github.com/helix-editor/helix/issues/3721 and https://github.com/helix-editor/helix/issues/312
